projects/media-viewer/src/lib/toolbar/main-toolbar/left-pane/left-pane.component.ts

```diff
@@ -20,6 +19,10 @@ export class ToolbarLeftPaneComponent implements OnInit {
     this.toolbarEvents.setCurrentPage.subscribe(pageNumber => this.setCurrentPage(pageNumber));
   }
 
+  public ngOnDestroy(): void {
+    this.toolbarEvents.setCurrentPage.unsubscribe();
```

Comment:
not sure this is going to work…. you need to call unsubscribe on the Subscription object the subscribe returned

Reply:
oh....